include/robot_localization/navsat_transform.h

```diff
@@ -87,11 +90,11 @@ class NavSatTransform
     //!
     bool fromLLCallback(robot_localization::FromLL::Request& request, robot_localization::FromLL::Response& response);
 
-    //! @brief Given the pose of the navsat sensor in the UTM frame, removes the offset from the vehicle's centroid
-    //! and returns the UTM-frame pose of said centroid.
+    //! @brief Given the pose of the navsat sensor in the cartesian frame, removes the offset from the vehicle's
+    //! centroid and returns the cartesian-frame pose of said centroid.
     //!
-    void getRobotOriginUtmPose(const tf2::Transform &gps_utm_pose,
-                               tf2::Transform &robot_utm_pose,
+    void getRobotOriginCartesianPose(const tf2::Transform &gps_cartesian_pose,
+                               tf2::Transform &robot_cartesian_pose,
```

Comment:
With the method name change, the second and third parameters are now misaligned. Would you mind updating that?

Reply:
Done.